Update CODE_OF_CONDUCT.md (#8052)
Kong/kong: DEVELOPER.md

[CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md), and [COPYRIGHT](COPYRIGHT) before
you start.

> If you are planning to develop on Kong, you'll need a development

Reviewer: This change is not needed, "on developing" is an acceptable use here.
```suggestion
If you are planning on developing on Kong, you'll need a development
```

#### Kong Gojira

[Gojira](https://github.com/Kong/gojira) is a CLI that uses docker-compose

> internally to make the necessary setup of containers for getting all
> dependencies required to run a particular branch of Kong locally, as well

Reviewer: This phrase could be further simplified:
```suggestion
to make several containers with all the dependencies 
required to run a particular branch of Kong locally, as well
```
